packages/tools/client-debugger/src/Registry.ts

```diff
@@ -45,6 +46,66 @@ export interface FluidClientDebuggerProps {
 	containerNickname?: string;
 }
 
+export interface DebuggerRegistryEvents extends IEvent {
```

Comment:
We should document both of these interfaces. We should also add the `@internal` tag to both of these, as I don't expect external (outside of our repo) consumers to use these types.